Fixed Merge conflicts. Checked lines from 897 to 1528.

39sStanislavskii opened the pull request:
Fixed Merge conflicts. Checked lines from 897 to 1528. Added missing lines for expoter, component.selector, inspector, also volume slicer and it's translations. Fixed some old texts and added other missing lines between 897 and 1528 lines (lines numbers according to en.json)

39sStanislavskii committed:
Restructure of old ru.json under new en.json

39sStanislavskii committed:
Lines 391-504 Account information, credentials

39sStanislavskii committed:
Merge remote-tracking branch 'upstream/main from panther'

39sStanislavskii committed:
Merge remote-tracking branch 'upstream/main'

39sStanislavskii committed:
Fixed Merge conflicts. Checked lines from 897 to 1528. Added missing lines for expoter, component.selector, inspector, also volume slicer and it's translations. Fixed some old texts and added other missing lines between 897 and 1528 lines (lines numbers according to en.json)

39sStanislavskii committed:
removed missed ==== line

shadowpanther commented on ru.json:
```diff
@@ -427,6 +427,38 @@
         "Register.UsernameAlreadyRegistered": "Имя пользователя уже используется",
         "Register.ErrorCheckingUserId": "Ошибка проверки UserID",
 
+        "Login.InvalidCredentials": "Неверные учетные данные",
+        "Login.InvalidLogin": "Неверные учетные данные",
+
+        "Login.MalformedUID": "Некорректная информация об устройстве",
+        "Login.MissingUID": "Отсутствует информация об устройстве",
+        "Register.MissingUID": "Отсутствует информация об устройстве",
+        "Register.MalformedUID": "Некорректная информация об устройстве",
+
+        "Login.MissingLogin": "Не указан логин",
+        "Login.InvalidRecoveryCode": "Неверный код восстановления",
+        "Login.UnsupportedAuthentication": "Неподдерживаемый метод аутентификации",
+        "Login.EmailNotVerified": "Электронная почта не подтверждена",
+        "Login.AccountTempLocked": "Ваш аккаунт временно заблокирован.",
+        "Login.AccountBanned": "Ваш аккаунт забанен.",
+        "Login.DuplicateAccount": "Аккаунт дубликат",
+
+        "Register.MissingRequest": "Отсутвуют данные для регистрации",
+        "Register.InvalidEmail": "Некорректная электронная почта",
+        "Register.MissingUsername": "Отсутвует имя пользователя",
+        "Register.MissingEmail": "Отсутвует электронная почта",
+        "Register.UsernameTooLong": "Имя пользователя слишком длинное",
+        "Register.UsernameContainsAmpersand": "В имение пользователя не должны присутвовать символы: @",
+        "Register.UsernameStartsWithU": "Имя пользователя не должно начинаться с U-",
+        "Register.InvalidUsername": "Некорректное имя пользователя",
+        "Register.InvalidDateOfBirth": "Некорректная дата рождения",
+        "Register.UserContainsBlockedTerms": "Имя пользователя содержит запрещенные слова",
+        "Register.MissingPassword": "Отсутствует пароль",
+        "Register.InvalidPassword": "Некорректный пароль",
+        "Register.EmailAlreadyRegistered": "Электронная почта уже используется",
+        "Register.UsernameAlreadyRegistered": "Имя пользователя уже используется",
+        "Register.ErrorCheckingUserId": "Проверка ошибок UserID",
+
```
Этот блок уже есть в переводе

shadowpanther commented on ru.json:
```diff
@@ -1336,12 +1386,17 @@
         "Inspector.Texture.RotateCCW": "Повернуть на 90° против ЧС",
         "Inspector.Texture.Rotate180": "Повернуть на 180°",
-
+        "Inspector.Texture.TrimTransparent" : "Обрамление прозрачности",
+        "Inspector.Texture.TrimByCornerColor" : "Обрамление по цвету углов",
```
Обрезать прозрачное и обрезать по цвету угла (это для текстур, похожие операции есть в Photoshop)

shadowpanther commented on ru.json:
```diff
@@ -1410,6 +1466,21 @@
+        "Inspector.SimpleAwayIndicator.TestAway" : "Выставить как 'отошел' (debug)",
+        "Inspector.SimpleAwayIndicator.TestRestore" : "Восстановить (debug)",
+        
+        "Inspector.Fields.Drive" : "Двигать",
+        "Inspector.Fields.DriveWriteBack" : "Двигать <size=50%>(с откликом)</size>",
```
Драйвить
Драйвить (с обратной записью)

Звучит коряво, но тут не стоит отходить от терминологии оригинала.

shadowpanther reviewed (changes requested):
Убери дублирующийся кусок (регистрация и логин) и поправь в двух местах согласно комментариям.

39sStanislavskii committed:
Fixed duplicate of already translated text. Fixed some translations. 

Inspector.Fields.Drive
Inspector.Fields.DriveWriteBack
Inspector.Texture.TrimTransparent
Inspector.Texture.TrimByCornerColor
